interfaces/builtin/removable_media.go

```diff
@@ -42,6 +42,11 @@ const removableMediaConnectedPlugAppArmor = `
 # Mount points could be in /run/media/<user>/* or /media/<user>/*
 /{,run/}media/*/ r,
 /{,run/}media/*/** rwk,
+
+# Allow read-only access to /mnt to enumerate items.
+/mnt/ r,
+# Allow write access to anything under /mnt
+/mnt/*/** rwk,
```

Comment:
Also assuming this is from an unrelated PR.

Reply:
Indeed.